//Exception handling

//Throw exceptions

try {
    throw "Exception!";
}catch(e){
    console.log(e);
}

console.log("I want to run");